src/@ionic-native/plugins/purchases/index.ts

```diff
@@ -2,6 +2,147 @@ import { Injectable } from '@angular/core';
 import { Plugin, Cordova, IonicNativePlugin } from '@ionic-native/core';
 import { Observable } from 'rxjs';
 
+/**
+ * @deprecated use ATTRIBUTION_NETWORK instead
+ *
+ * Enum for attribution networks
+ * @readonly
+ * @enum {Number}
+ */
+export enum ATTRIBUTION_NETWORKS {
+  APPLE_SEARCH_ADS = 0,
+  ADJUST = 1,
+  APPSFLYER = 2,
+  BRANCH = 3,
+  TENJIN = 4,
+  FACEBOOK = 5,
+}
+
+export enum ATTRIBUTION_NETWORK {
+  APPLE_SEARCH_ADS = 0,
+  ADJUST = 1,
+  APPSFLYER = 2,
+  BRANCH = 3,
+  TENJIN = 4,
+  FACEBOOK = 5,
+}
+
+export enum PURCHASE_TYPE {
+  /**
+   * A type of SKU for in-app products.
+   */
+  INAPP = 'inapp',
+
+  /**
+   * A type of SKU for subscriptions.
+   */
+  SUBS = 'subs',
+}
+
+/**
+ * @deprecated use PURCHASE_TYPE instead
```

Comment:
`ProductType` was in this wrapper before and I deprecated it in favor of `PURCHASE_TYPE` which is the same as in the cordova plugin